common/rdm/PidStore.cpp

```diff
@@ -208,6 +208,17 @@ bool PidDescriptor::IsSetValid(uint16_t sub_device) const {
 }
 
 
+/**
+ * @brief Compare PIDs by name
+ *
+ * Suitable for use as a sort comparison function
+ */
+bool PidDescriptor::CompareByName(const PidDescriptor *a,
```

Comment:
I did wonder about a more consistent name, but we currently have a mix of lt and LessThan suffixes and prefixes as well as ordering, and snake and camel case, so one for another day I think! Although if you wanted to touch it, perhaps order/ordering rather than compare given it's not the cmp operator underneath.